AirBench/Models/User.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace AirBench.Models
+{
+    public class User
+    {
+        public User() { }              //empty constructor
+
+        //initializing constructor
+        public User(int id, string name, string userName, string password)
```

Comment:
You didn't include `id` in your other model constructors, so for consistency, I'd remove that parameter here.